doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: page
+title: Cassandra log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender
+sort_idx: 10
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* TOC
+{:toc}
+
+The Cassandra log appender is responsible for transferring logs from the Operations server to the Cassandra database.
+
+# Creating Cassandra log appender with Admin UI
+
+The easiest way to create a Cassandra log appender for your application is by using Admin UI.
+
+To create a log appender do the following:
+
+1. In the <b>Log appenders</b> window, click <b>Add log appender</b>.
+2. Enter the log appender name and description, select the minimum and maximum supported schema version, select necessary log metadata fields.
+3. Set the log appender type to <i>Cassandra</i>.
+4. Fill in the Cassandra log appender configuration form.
+5. Click <b>Add</b>.
+
+<img src="attach/create-cassandra-log-appender-admin-ui.png">
+
+# Creating Cassandra log appender with Admin REST API
+
+It is also possible to create a Cassandra log appender instance by using [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs #TODO).
+The following example illustrates how to provision the Cassandra log appender via Admin REST API.
+
+## Configuration
+
+The Cassandra log appender configuration should match to
+[this](https://github.com/kaaproject/kaa/blob/master/server/appenders/cassandra-appender/src/main/avro/cassandra-appender-config.avsc) Avro schema.
+
+* Cassandra nodes - list of Cassandra hosts.
+* Authentication credentials - credentials used to authenticate on Cassandra cluster
+* Keyspace name – Cassandra keyspace used to prefix the data table
+* Table name pattern – pattern used to create table name (ex.: <b>logs_$app_token</b> adds the application token at the end of the table name).
+* Column Mapping - section that handles data mapping configuration. It can map specific log data to appropriate columns.
+
+|Type |Example value|Example column type|Description|
+|---||---||---||---|
+|${TYPE}_BINARY||BLOB|Maps one of the fields listed above in binary format|
+|${TYPE}_JSON||TEXT|Maps one of the fields listed above as a corresponding JSON|
+|CLIENT_FIELD|clientField|TEXT|Maps a client-side endpoint profile field to the specified column|
+|EVENT_FIELD |telemetry|DOUBLE|Maps a log schema field to the specified column|
+|HEADER_FIELD|endpointKeyHash, applicationToken, headerVersion,timestamp, logSchemaVersion|TEXT|Maps a header variable to specified the column|
+|SERVER_FIELD|serverField|TEXT|Maps a server-side endpoint profile field to the specified column|
+|TS|ddMMyy|BIGINT|Maps a timestamp generated by Kaa to the specified field. Timestamp pattern is handled by java.text.SimpleDateFormat|
+|UUID|(empty)|UUID|Maps a UUID generated by Kaa to the specified field|
```

Comment:
In one case example value is set to "(empty)" and in other - "". What is the difference?